internal/persistence/persistors/mysqlstore/store_organization.go

```diff
@@ -0,0 +1,290 @@
+				mysqlmodel.TableNames.Organization,
+				mysqlmodel.OrganizationColumns.LastUpdatedAt,
+				mysqlmodel.OrganizationColumns.LastUpdatedAt,
+			),
+			fmt.Sprintf("%s.%s AS %s",
+				mysqlmodel.TableNames.Organization,
+				mysqlmodel.OrganizationColumns.IsActive,
+				mysqlmodel.OrganizationColumns.IsActive,
+			),
+		),
+	}
+
+	if filters != nil {
+		if len(filters.IdsIn) > 0 {
+			queryMods = append(queryMods, mysqlmodel.OrganizationWhere.ID.IN(filters.IdsIn))
+		}
+
+		if filters.OrganizationIsActive.Valid {
+			queryMods = append(queryMods, mysqlmodel.OrganizationWhere.IsActive.EQ(filters.OrganizationIsActive.Bool))
+		}
+
+		if len(filters.OrganizationNameIn) > 0 {
+			queryMods = append(queryMods, mysqlmodel.OrganizationWhere.Name.IN(filters.OrganizationNameIn))
+		}
+
+		if filters.CreatedBy.Valid {
+			queryMods = append(queryMods, mysqlmodel.OrganizationWhere.CreatedBy.EQ(filters.CreatedBy))
+		}
+
+		if filters.LastUpdatedBy.Valid {
+			queryMods = append(queryMods, mysqlmodel.OrganizationWhere.LastUpdatedBy.EQ(filters.LastUpdatedBy))
+		}
+	}
+
+	q := mysqlmodel.Organizations(queryMods...)
+	totalCount, err := q.Count(ctx, ctxExec)
+	if err != nil {
+		return nil, fmt.Errorf("get organizations count: %w", err)
+	}
+
+	page := pagination.Page
+	maxRows := pagination.MaxRows
+	if filters != nil {
+		if filters.Page.Valid {
+			page = filters.Page.Int
+		}
+		if filters.MaxRows.Valid {
+			maxRows = filters.MaxRows.Int
+		}
+	}
+
+	pagination.SetQueryBoundaries(page, maxRows, int(totalCount))
+
+	queryMods = append(queryMods, qm.Limit(pagination.MaxRows), qm.Offset(pagination.Offset))
+	q = mysqlmodel.Organizations(queryMods...)
+
+	if err = q.Bind(ctx, ctxExec, &res); err != nil {
+		return nil, fmt.Errorf("get organizations: %w", err)
+	}
+
+	pagination.RowCount = len(res)
+	paginated.Organizations = res
+	paginated.Pagination = pagination
+
+	return &paginated, nil
+}
+
+func (m *Repository) CreateOrganization(ctx context.Context, tx persistence.TransactionHandler, organization *model.Organization) (*model.Organization, error) {
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return nil, fmt.Errorf("extract context executor: %w", err)
+	}
+
+	entry := mysqlmodel.Organization{
+		Name: organization.Name,
+	}
+	if err = entry.Insert(ctx, ctxExec, boil.Infer()); err != nil {
+		return nil, fmt.Errorf("insert organization: %w", err)
+	}
+
+	organization, err = m.GetOrganizationById(ctx, tx, entry.ID)
+	if err != nil {
+		return nil, fmt.Errorf("get organization by id: %w", err)
+	}
+
+	return organization, nil
+}
+
+func (m *Repository) AddOrganization(ctx context.Context, tx persistence.TransactionHandler, organization *model.Organization) (*model.Organization, error) {
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return nil, fmt.Errorf("extract context executor: %w", err)
+	}
+
+	entry := &mysqlmodel.Organization{
+		Name:      organization.Name,
+		CreatedBy: organization.CreatedBy,
+	}
```

Comment:
_organization.CreatedBy_: ... Is this intended to be an int column? If yes, it shouldn't be. It should be a string, and contains the full name of the person who created this organization.

Remember, we are passing userId int, as a new argument in whatever create/add organization function we are creating, and we use that.

Reply:
So having said that, this change will definitely affect your tests.

Reply:
So is it ok if we change your migration file? you set the createdby as int in the migration file.

Reply:
But in the app Model (not mysqlmodel) for organization, make the createdby, and updatedby STRINGS.. that contain the full name of the user

Reply:
so yes, still has test implications

Reply:
Alright on it. Also can you give us feedback regarding the mutations issue?

Reply:
uhh what mutations issue are we talking about?

Reply:
I talked to demby regarding this and he mentioned that all of the creation of structs must happen inside of the mutations as to not pollute the test file. Static data is alright for testing purposes and if it can be done, fields with numbers should be dynamic.

I've pushed and updated the get orgs func. The update has made the function and the test cases much cleaner now.